src/hotspot/share/gc/g1/heapRegion.inline.hpp

```diff
@@ -200,7 +200,6 @@ inline void HeapRegion::reset_compacted_after_full_gc() {
 
 inline void HeapRegion::reset_pinned_after_full_gc() {
```

Comment:
Maybe rename the method since it now does not only apply to "pinned" regions, but any region that is not compacted.
E.g. `reset_not_compacted_after_full_gc()`.